EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1597,6 +1597,10 @@ void V2::World::output(const mappers::VersionParser& versionParser) const
 	outputCultures();
 	Log(LogLevel::Progress) << "96 %";
 
+	LOG(LogLevel::Info) << "<- Writing Religion Definitions";
+	outputReligions();
+	Log(LogLevel::Progress) << "96 %";
```

Comment:
Just dump this line, or shift all previous ones by one until you manage to make room.